src/main/java/mate/academy/mybookshop/dto/UpdateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.mybookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class UpdateBookRequestDto {
```

Comment:
I think we do not need another DTO with the same fields when we can reuse the existing `create` one and rename it to something more general like `BookRequestDto`